crates/tests/interface/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "test_interface"
```

Comment:
Currently, if you rely on nightly features (like the `implement` macro) then you need to name your test crate a little differently so that the yml generator groups it acccordingly:

https://github.com/microsoft/windows-rs/blob/324a0c082f2f6019840ad4e30d4920682c0637f9/crates/tools/yml/src/main.rs#L241-L243

I should just make this "nightly" instead of "implement" but that should unblock this for now.

Reply:
Unfortunately, `tool_yml` exists because things like `cargo test --all` will just run out of memory on GitHub's build servers. So that tool unrolls the `--all` flag.